lib/associations/has-many.js

```diff
@@ -115,19 +115,19 @@ var HasMany = function(source, target, options) {
     /**
      * Associate several instances with this.
      *
-     * @param {Array<Instance|String|Number>} [newAssociations] An array of instances or primary key of instances to associate with this. Pass `null` or `undefined` to remove all associations.
+     * @param {Array<Instance|String|Number>} [newAssociations] An array of instances or primary key of instances to associate with this.
      * @param {Object} [options] Options passed to `target.update`.
-     * @param {Object} [options.validate] Run validation for the join model
+     * @param {Object} [options.validate] Run validation for the join model.
      * @return {Promise}
      * @method addAssociations
      */
     addMultiple: 'add' + plural,
     /**
-     * Associate several instances with this.
+     * Associate one instance with this.
```

Comment:
Hi @mickhansen, I believe this particular JSDoc belongs to `add`, not `addMultiple`.
Do you actually mean that the singular `add` will add multiple instances as well?